app/uk/gov/hmrc/teamsandrepositories/connectors/GithubConnector.scala

```diff
@@ -67,7 +67,7 @@ class GithubConnector @Inject()(
       .setHeader(authHeader)
       .withProxy
       .execute[List[TeamWithPermission]]
-      .map(_.filter(_.permission == "push").map(_.name))
+      .map(_.filterNot(team => team.permission == "pull" || team.permission == "triage").map(_.name))
```

Comment:
`.map(_.filterNot(team => Seq("pull", "triage").contains(team.permission)).map(_.name))` might scale better when maintaining the list

Reply:
I've updated it now